sections/04_study.md

```diff
@@ -670,7 +670,9 @@ several percentage points of improvement compared to random forests or deep
 networks trained only on CpG or sequence information. Similar deep learning
 methods have been applied to impute low-resolution ChIP-seq signal from bulk
 tissue with great success, and they could easily be adapted to single cell data
-[@tag:Qin2017_onehot @tag:Koh2016_denoising].
+[@tag:Qin2017_onehot @tag:Koh2016_denoising]. Deep learning has also been useful
+for dealing with batch effects, which can severely affect single-cell data
```

Comment:
Ahh - after another glance - I see what you've done here. I was interpreting the reference as related to the ", which" clause.